### ISSUE TITLE
Training dataset and Testing dataset

### ISSUE THREAD

**Issue by Arusa1** [tengyu1998/SCI#21]
Excellent work!
There are two main questions for me after reading the paper:
1. What is the training data of SCI? To my opinion, zero-reference learning methods highly depend on the disign of loss function and the training data. However, the paper seems to make no mention of this, nor does the supplementary material.
2. What is the specific testing data of SCI? It is mentioned that SCI randomly sampled 100 images from MIT dataset and 50 testing image from LSRW dataset for testing. So could you please release these images to help reproduce the great results in the paper?

Looking forward to your reply. Thanks!

**Comment by peylnog** [comment]
+1

**Comment by McFly-Jia** [comment]
+1

**Comment by JackHe1999** [comment]
+1

**Comment by mv-lab** [comment]
+1

**Comment by Magic-Koala** [comment]
I also want to know what is the training data of SCI?

**Comment by hzy-del** [comment]
> Excellent work! There are two main questions for me after reading the paper:
> 
> 1. What is the training data of SCI? To my opinion, zero-reference learning methods highly depend on the disign of loss function and the training data. However, the paper seems to make no mention of this, nor does the supplementary material.
> 2. What is the specific testing data of SCI? It is mentioned that SCI randomly sampled 100 images from MIT dataset and 50 testing image from LSRW dataset for testing. So could you please release these images to help reproduce the great results in the paper?
> 
> Looking forward to your reply. Thanks!

Hello, do you know the composition of the training data set?

**Comment by Arusa1** [comment]
> > Excellent work! There are two main questions for me after reading the paper:
> > 
> > 1. What is the training data of SCI? To my opinion, zero-reference learning methods highly depend on the disign of loss function and the training data. However, the paper seems to make no mention of this, nor does the supplementary material.
> > 2. What is the specific testing data of SCI? It is mentioned that SCI randomly sampled 100 images from MIT dataset and 50 testing image from LSRW dataset for testing. So could you please release these images to help reproduce the great results in the paper?
> > 
> > Looking forward to your reply. Thanks!
> 
> Hello, do you know the composition of the training data set?

I can only find a vague discription of the dataset used in the paper in [https://github.com/vis-opt-group/SCI](url). However, it is still unclear exactly what training data set and testing data the authors used. The discription in the readme.md used the words like "mainly", "randomly", etc.
